_plugins/dashboard.rb

```diff
@@ -43,6 +43,13 @@ def self.munge_project_data(project_data)
       FIELDS_TO_TRANSLATE.each do |from, to|
         project_data[to] = project_data[from] unless project_data[to]
       end
+      original_name = project_data['name']
+      if original_name.scan(/[A-Z]/).size > 0
+        project_data['redirect_from'] = Array.new
+        project_data['redirect_from'].push("project/#{original_name}")
```

Comment:
If there's a world where we get [redirect codes from the Team API](https://github.com/18F/team-api.18f.gov/issues/146) there could be more than one project in the `redirect_from` key. If we got a 301 on a project name, for example, we would add the old `name` to the `redirect_from` key on the new `name`.